frontend/www/js/omegaup/components/user/Rank.test.ts

```diff
@@ -0,0 +1,58 @@
+import { shallowMount, mount } from '@vue/test-utils';
+
+import T from '../../lang';
+
+import user_Rank from './Rank.vue';
+
+describe('Rank.vue', () => {
+  const propsData = {
+    page: 1,
+    length: 10,
+    isIndex: true,
+    isLogged: true,
+    availableFilters: {},
+    filter: '',
+    ranking: [],
+    resultTotal: 0,
+    pagerItems: [
+      {
+        class: 'disabled',
+        label: '1',
+        page: 1,
+      },
+    ],
+    searchResultUsers: [],
+  };
+
+  it('Should handle an empty rank', () => {
+    const wrapper = shallowMount(user_Rank, {
+      propsData,
+    });
+
+    expect(wrapper.find('table').exists()).toBe(false);
```

Comment:
Ya existe una función que espera un valor falso:

```suggestion
    expect(wrapper.find('table').exists()).toBeFalsy();
```